mantidproject/mantid
# Add instrument definition for NEAT at HZB

**Description of work.**

Users are currently copying the facilities/instrument definition to their mantid installations. Shipping them will make it easier to access them from a standard installation.

**Report to:** Spencer

**To test:**

System test checks should pass.
Start MantidPlot or Workbench and check that there is a `NEAT` instrument under the `HZB` facility.

*There is no associated issue.*

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
